asrtoolkit/combine_audio_files.py

```diff
@@ -0,0 +1,93 @@
+#!/usr/bin/env python
+"""
+Script for splitting audio files using a transcript with start/stop times
+"""
+import argparse
+import logging
+import os
+import operator
+from functools import reduce
+
+from asrtoolkit.data_structures.time_aligned_text import time_aligned_text
+from asrtoolkit.data_structures.audio_file import audio_file, combine_audio
+from asrtoolkit.file_utils.script_input_validation import valid_input_file
+from asrtoolkit.file_utils.name_cleaners import strip_extension
+
+LOGGER = logging.getLogger(__name__)
+
+def check_transcript(transcript):
+    if valid_input_file(transcript):
+        return time_aligned_text(input_data=transcript)
+    else:
+        LOGGER.error("Invalid transcript file {}".format(transcript))
+
+
+def check_audio_file(audio_file_name):
+    if valid_input_file(audio_file_name, ["mp3", "sph", "wav", "au", "raw"]):
+        return audio_file(audio_file_name)
+    else:
+        LOGGER.error("Invalid audio file {}".format(audio_file_name))
+
+
+def check_transcript_segment(segment):
+    if not hasattr(segment, 'start'):
+        LOGGER.error("Transcript segment doesn't include the start time, segment: {}".format(segment))
```

Comment:
Should this function throw an exception or return False to stop the workflow, or will the script still work if there are no start times?

Reply:
You're right, this script should exit on those errors! Thanks.

Fixed in error handling commit.